v1.0/CommandLineTool.yml

```diff
@@ -787,6 +796,13 @@ $graph:
         May be an expression.  If so, the expression return value must validate
         as `{type: array, items: [File, Directory]}`.
 
+        Files or Directories which are listed in the input parameters and
+        appear in the `InitialWorkDirRequirement` listing must have their
+        `path` set to their staged location in the designated output directory.
```

Comment:
I'm fine with this, but this is in contradiction with #396 where Directory in InitialWorkdirRequirement lacks path (which is also a reason why bunny fails that test).